bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrole-node-crd-reader.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:openshift:controller:check-endpoints-node-crd-reader
```

Comment:
after this we have the old and th e new object, with overlap. Why not add one for CRDs only?

Reply:
These are new for v4.6.